src/test/java/org/folio/edge/rtac/utils/RtacUtilTest.java

```diff
@@ -0,0 +1,81 @@
+package org.folio.edge.rtac.utils;
+
+import static io.vertx.core.http.HttpHeaders.ACCEPT;
+import static org.folio.edge.core.Constants.TEXT_PLAIN;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import io.vertx.core.MultiMap;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+
+@RunWith(VertxUnitRunner.class)
+public class RtacUtilTest {
+
+  @Mock
+  private static HttpServerRequest rtacRequest;
+  @Mock
+  private static MultiMap rtacHeaders;
+
+  private static List<String> supportedAcceptHeaders = Arrays
+      .asList(RtacMimeTypeEnum.getAllTypesAsString());
+  private static List<String> unsupportedAcceptHeaders = Arrays.asList(TEXT_PLAIN);
+
+  @BeforeClass
+  public static void beforeClassRtacUtils() {
+    // Creates mock of RTAC request nd headers
+    rtacRequest = mock(HttpServerRequest.class);
+    rtacHeaders = mock(MultiMap.class);
```

Comment:
Could a real request or map be used for these instead of mocks?

Reply:
I believe we can't use RestAssured approach here to extract request object, to be honest I didn't find how to do this easy. because we can only define request via RequestSpecification there or can deal with real response when invocation will be completed. But possibly it doesn't matter if we use real request on fake, because these mocks only needed for utility class testing, so the needs just to get any request and then checks it. Therefore mocking works here fine, as for me. Whereas the RtacHandlerTest uses real requests.

Reply:
> I believe we can't use RestAssured approach here to extract request object, to be honest I didn't find how to do this easy.

As far as I can tell, this is a unit test, RestAssured wouldn't be appropriate for that.

By real requests, I mean real instances of `HttpServerRequest` or at least `MultiMap` rather than a mock.

I was wondering why the tests don't use real collaborators.

Reply:
Thanks for pointing this. Really by replacing mocking of headers by real impl all the code related to mocking headers behavior will be omitted. I'll fix this. Thank you, Marc.